### DISCUSSION TITLE
Will this work on Nextion 3.5" 480x320 display?

### DISCUSSION THREAD

**Discussion by zraken** [Blackymas/NSPanel_HA_Blueprint#234] [General]
I would like to know if the HMI code will work as-is on a 3.5" 480x320 Nextion LCD panel (NX4832K035). I plan on hooking it up with an ESP32 loaded with ESPHome. Looks like the NSPanel resolution is the same, the only difference might be the lack of buttons.
I will need to update nspanel_esphome.yaml for correct tx/rx pins and remove things like the buzzer, adc etc.
Can this be done?

**Comment by Blackymas** [comment]
hi,

I think yes but not tested :)

**Reply by zraken** [reply]
Ok, I'll give it a shot and report back.
